libafl_bolts/src/tuples.rs

```diff
@@ -731,6 +731,43 @@ where
     }
 }
 
+/// Trait for structs which are capable of mapping a given type to another.
+pub trait Mapper<T> {
```

Comment:
`Mappable`?

Reply:
No, this is the mapping operation. This has to be a separate trait as multiple types are converted, potentially to a different type according to the particular mapping operation taking place.

Reply:
`transform`?